internal/runners/commit/commit.go

```diff
@@ -40,34 +40,34 @@ func New(p primeable) *Stage {
 	}
 }
 
-func (s *Stage) Run() error {
+func (s *Commit) Run() error {
```

Comment:
Please rename the receiver to either `c` (for "commit") or `r` (for "runner" - far less preferable in my opinion).

Reply:
Thanks for pointing this out.